api/v1alpha1/groupversion_info.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2022.
```

Comment:
Why year is being changed?
It was created in 2022 so copyright is from that year

Reply:
ok, will go back and change, thanks